modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/TransactionProxyImpl.java

```diff
@@ -421,4 +421,63 @@ private void saveFuture(IgniteInternalFuture<IgniteInternalTx> fut) {
     @Override public String toString() {
         return S.toString(TransactionProxyImpl.class, this);
     }
+
+    /** {@inheritDoc} */
+    @Override public void savepoint(String name) throws IllegalArgumentException {
+        savepoint(name, false);
```

Comment:
From ticket description : "If the current transaction has a savepoint with the same name, the old savepoint is deleted and a new one is set"
So , by default , overwrite must be true